addon/client.go

```diff
@@ -183,7 +189,7 @@ func (r *Client) Delete(path string) (err error) {
 			Method: http.MethodDelete,
 			URL:    r.join(path),
 		}
-		request.Header.Set(auth.Header, r.token)
+		request.Header.Set("Accept", "")
```

Comment:
Assuming intention not to mix the const with a string value.

Reply:
good catch.  Nope just missed this spot.